packages/core/src/components/Label.tsx

```diff
@@ -2,7 +2,7 @@ import React, { FC } from "react";
 import { LabelVariant } from "../index";
 import classNames from "classnames";
 
-interface Props {
+export interface LabelProps {
```

Comment:
Tyvstartet på featureønsket ditt @HavardPede 😄 Fikk bruk for det internt i Jøkul også.

Reply:
Ja jeg skal hive meg på den oppgaven når det har roet seg litt med det prosjektet jeg står i.